src/libmongoc/src/mongoc/mongoc-topology-private.h

```diff
@@ -37,6 +37,15 @@
 #define MONGOC_TOPOLOGY_HEARTBEAT_FREQUENCY_MS_SINGLE_THREADED 60000
 #define MONGOC_TOPOLOGY_MIN_RESCAN_SRV_INTERVAL_MS 60000
 
+/*
```

Comment:
Though we're only using this in libmongoc now, I could see this being useful in libbson too. libmongoc can't reach inside libbson's private headers. But I like what you've done by creating a common-config.h.in. I'd argue for creating another common header, `common-macros.h`, and placing this inside.